Dockerfile

```diff
@@ -2,15 +2,13 @@ FROM node:14-slim
 
 WORKDIR /app
 
-# Install dependencies, but don't generate lock files as we want to always fetch
-# the lastest @mdn/browser-compat-data package.
 COPY package.json ./
-RUN npm install --no-package-lock && \
-  npm cache clear --force
+RUN npm install --no-package-lock; \
+  npm cache clean --force
 
 # copy sources
 COPY update_remote_settings_records.mjs ./
 COPY ./version.json ./
 
 # set CMD
-CMD ["npm", "run", "ingest"]
+CMD ["node", "update_remote_settings_records.mjs"]
```

Comment:
We are no longer using `ingest`, so I'm not sure if we should keep it in the package.json?

Reply:
oh, good catch, forgot to clean that file up .. will fix this with another PR